signalfx/resource_signalfx_aws_external_integration.go

```diff
@@ -0,0 +1,119 @@
+package signalfx
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/signalfx/signalfx-go/integration"
+)
+
+func integrationAWSExternalResource() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Name of the integration",
+			},
+			"external_id": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Sensitive:   true,
+				Description: "The SignalFx-generated AWS account ID to use with an AWS integration.",
```

Comment:
isn't the SignalFx _AWS account_ ID different than the _external_ ID?

Reply:
Yup!